### DISCUSSION TITLE
Is there a way to update other data fields related to the job?

### DISCUSSION THREAD

**Discussion by Ben-CA** [OptimalBits/bull#2470] [Q&A] [answered]
So we can update the job **data** with something like this:

`job.update({...job.data, "failed": true, "dateFailed": new Date()})`

Is there a way we can update other Redis keys related to the job? So for instance, if there's an error, and I wanted to update the returnedvalue with a message / status?

Screenshot of a job visualized in Redis Commander:

![image](https://user-images.githubusercontent.com/47060874/202264890-33e0f05a-03b7-4955-961f-ad8c8c4420bb.png)

**Comment by Ben-CA** [comment, marked as answer]
Actually, for my case it looks like I can use the "failedReason" field.

**Reply by manast** [reply]
It is only possible to update the data field, the rest cannot be changed after the job has been added and we are not going to change this behavior as it would open a can of worms of possible edge cases.

**Reply by Ben-CA** [reply]
Makes sense, thanks!
